### ISSUE TITLE
Scrobble doesnt work

### ISSUE THREAD

**Issue by SayWhat69** [stigger/trakt-for-appletv#18]
When starting playback with for example Netflix, I get a message, but it doesnt contain the Playing Title:
`playbackState: Playing
playerPath {
  origin {
    displayName: "XXXXXXXX"
    identifier: XXXXXXXXX
  }
  client {
    processIdentifier: XXX
    bundleIdentifier: "com.netflix.Netflix"
  }
  player {
    identifier: "avkit-XXXXXXX-XXXX-XXXX-XXXX-XXXXXXXXXXXXX"
    displayName: "player-2"
  }
}
playbackStateTimestamp: 628599540.6316`

Since it doesnt contain "playbackQueue" it doesnt go past the line 
`if len(state_msg.playbackQueue.contentItems) > 0:`

Do I have to setup anything on the Apple TV to send the Playing Title?

**Comment by stigger** [comment]
No, you don't need to setup anything on Apple TV. The only caveat is that it might not work correctly if you're launching the scrobbler after the playback has already started on Apple TV. If that's not the case, then something might've changed, but I cannot verify this because I don't have a Netflix subscription at the moment. Can you check if the scrobbling works correctly for you with Apple TV+ or purchased tv shows?

**Comment by SayWhat69** [comment]
Tried it with Apple TV+ - Same Result.

Maybe its not compatible with the new tvOS 14?

**Comment by stigger** [comment]
No, it is, I'm using latest tvOS and everything works fine for me.

**Comment by SayWhat69** [comment]
Hm thats weird...

**Comment by louisdb03** [comment]
Maybe a stupid remark but for me, it looks like it doesn't work using the tv remote. When using an iPhone as tv remote it does recognize it. Maybe try launching your video using your iPhone as the remote.

You can know the apple tv is sharing its state over network when your iPhone shows the 'Now playing' widget on the lock screen.
